test/cmd/admin.sh

```diff
@@ -416,8 +417,8 @@ os::cmd::expect_success 'oc adm prune auth user/cascaded-user'
 os::cmd::expect_success 'oc delete user  cascaded-user'
 os::cmd::expect_success 'oc delete user  orphaned-user  --cascade=false'
 # Verify all identities remain
-os::cmd::expect_success 'oc get identities/anypassword:cascaded-user'
-os::cmd::expect_success 'oc get identities/anypassword:orphaned-user'
+os::cmd::expect_success 'oc get identities/alwaysallow:cascaded-user'
+os::cmd::expect_success 'oc get identities/alwaysallow:orphaned-user'
```

Comment:
Why this change?

Reply:
Same reason as above. For some reason I do not have anypassword: identities on localup master.